docs/translating.md

```diff
@@ -207,6 +207,13 @@ These files are uploaded to [Transifex][], where they can be translated.
 
 ## Backend translations
 
+A string in python can be marked for translation using the `_()` function,
```

Comment:
Also, I think it would be clearer to put this before:

> Zulip expects all the error messages...

So it doesn't get mixed up with the template stuff.